openmc/surface.py

```diff
@@ -185,22 +184,22 @@ class Plane(Surface):
 
     Parameters
     ----------
-    surface_id : int
+    surface_id : int, optional
         Unique identifier for the surface. If not specified, an identifier will
         automatically be assigned.
-    boundary_type : {'transmission, 'vacuum', 'reflective', 'periodic'}
+    boundary_type : {'transmission, 'vacuum', 'reflective', 'periodic'}, optional
```

Comment:
I have a branch with periodic BCs implemented actually. I can PR it sometime soon so I'll just leave these docstrings as is.